.github/workflows/dapr-base-containers.yaml

```diff
@@ -45,17 +53,51 @@ jobs:
         with:
           repository: ${{ env.CHECKOUT_REPO }}
           ref: ${{ env.CHECKOUT_REF }}
-      - name: docker login
-        if: env.CHECKOUT_REPO != ''
+      - name: Set REPO_OWNER
+        shell: bash
+        run: |
+          REPO_OWNER=${{ github.repository_owner }}
+          # Lowercase the value
+          echo "REPO_OWNER=${REPO_OWNER,,}" >>${GITHUB_ENV}
```

Comment:
Definitely not needed in `dapr` but can be necessary in forks, such as `ItalyPaleAle`'s :)